lib/pathUtils.js

```diff
@@ -183,3 +183,32 @@ const cachedJoin = (rootPath, request) => {
 	return cacheEntry;
 };
 exports.cachedJoin = cachedJoin;
+
+const checkExportsFieldTarget = relativePath => {
```

Comment:
I decided to write utility function for this for performance reasons. (checking "is request out of package scope" after normalization more expensive)